src/Microsoft.Data/StringColumn.BinaryOperations.cs

```diff
@@ -42,9 +42,8 @@ public override BaseColumn Add<T>(T value, bool inPlace = false)
             return ret;
         }
 
-        public override PrimitiveColumn<bool> Equals(BaseColumn column)
+        public override PrimitiveColumn<bool> ElementwiseEquals(BaseColumn column)
```

Comment:
Does the ArrowStringColumn not have this?

Reply:
Not yet. Filed https://github.com/dotnet/corefxlab/issues/2754